### DISCUSSION TITLE
More Auxiliary Units

### DISCUSSION THREAD

**Discussion by thomasherena** [Keriew/augustus#528] [Ideas]
This is definitely something that could be done as low priority, if it's even possible, but it might be cool to have more types of auxiliary units? The units would be based on the terrain type you have. For instance, if you were in the Northern territories, you might be able to recruit Gothic, Gaulish, Celtic, or Briton units, while if you're in the Central provinces, you could recruit Greek, Macedonian, or Etruscan units. Each unit would be one of the enemy units, but just retextured red, and they would require weapons to train. They'd be a cut-above the default auxiliaries, but not as strong as legionaries. Or perhaps each unit might have the same stats as their default "enemy" counterparts. 

Another idea might be having a sort of "riot-squad" fort. It would be just like a military fort and it would essentially funnel manpower from prefects to produce standing "armies" of prefects or sentries. They would be as powerful as their vanilla walker counterparts, but could be commanded directly, like any other legion. You would be allowed to build, at most, three. 

I'm not sure /how/ stats really work for combat, or if this is even possible but it might be a fun idea.

**Comment by MSTVD** [comment]
The first idea is something we already have plans for in the military update down the line, yes.

The idea for a type of "militia" fort is a concept i pitched and something that we might work on for 4.0, a sentry fort that is cheaper than a regular fort but you can only have one of as an early form of cheap but not super strong defense

**Reply by thomasherena** [reply]
Oooh, what kind of units would it hold? I assume it's just the sentry walkers?

**Reply by MSTVD** [reply]
Correct, watchmen/sentries. They'd fit nice aesthetically as a controllable unit as they look very military appropriate
